src/module.c

```diff
@@ -12447,7 +12447,7 @@ sds genModulesInfoString(sds info) {
  * -------------------------------------------------------------------------- */
 	 
 /* Check if the configuration name is already registered */
-int isModuleConfigNameRegistered(RedisModule *module, sds name) {
+int isModuleConfigNameRegistered(RedisModule *module, const char* name) {
```

Comment:
```suggestion
int isModuleConfigNameRegistered(RedisModule *module, const char *name) {
```
By convention the * sticks with the name.